signalfx-tracing/signalfx-dotnet-tracing/aspnetcore-and-mongodb/docker-compose.yml

```diff
@@ -5,20 +5,22 @@ services:
     ports:
      - "5000:5000"
     environment:
-      SIGNALFX_SERVICE_NAME: 'asp.net-core'
-      SIGNALFX_ENDPOINT_URL: 'http://smart-gateway:9080/v1/trace'
+      - SIGNALFX_SERVICE_NAME=asp.net-core
+      - SIGNALFX_ENDPOINT_URL=http://otel-collector:9411/api/v2/spans
+      - SIGNALFX_ENV
     depends_on:
       - client
       - mongo
-      - smart-gateway
+      - otel-collector
     user: 'root'
   client:
     build: src/ClientExample
     environment:
-      SIGNALFX_SERVICE_NAME: 'example-client'
-      SIGNALFX_ENDPOINT_URL: 'http://smart-gateway:9080/v1/trace'
+      - SIGNALFX_SERVICE_NAME=example-client
+      - SIGNALFX_ENDPOINT_URL=http://otel-collector:9411/api/v2/spans
+      - SIGNALFX_ENV
```

Comment:
Better to assign some value?

Reply:
Also a good idea: I listed just so I remember to use an `.env` file with the actual value but adding a value here it makes easy for people to try it directly with backend.